orangecontrib/text/widgets/owcorpusviewer.py

```diff
@@ -1,13 +1,14 @@
-from PyQt4 import QtGui
+from itertools import chain
+
 from PyQt4 import QtCore
-from PyQt4.QtGui import *
+from PyQt4 import QtGui
```

Comment:
Not even opposing, just asking. :stuck_out_tongue: I was also wrong. Looks good to me.